package.json

```diff
@@ -58,20 +59,22 @@
         "gh-pages": "^3.2.1",
         "html-webpack-plugin": "3.2.0",
         "husky": "1.3.1",
+        "identity-obj-proxy": "^3.0.0",
```

Comment:
This was recommended by Jest for mocking CSS modules in case we ever want to do snapshot testing, more details here https://github.com/keyz/identity-obj-proxy#real-world-example-wait-what-does-that-even-mean